lib/versioning/gradle/index.ts

```diff
@@ -35,38 +35,42 @@ const getMajor = (version: string): number | null => {
 const getMinor = (version: string): number | null => {
   if (isVersion(version)) {
     const tokens = tokenize(version.replace(regEx(/^v/i), ''));
```

Comment:
```suggestion
    const tokens = tokenize(version.replace(regEx(/^v/i), ''))!;
```

Workaround for now, we can fix later.

Alternative solution, extract body from `isVersion` to `valid` and return `tokens` or `null` there, reuse that parsed value in all other functions instead calling `isVersion` and do tokenizing twice

Reply:
Will try the alternative one 👍